docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Domain model & relationships
```

Comment:
```suggestion
title: Domain model and relationships
```

Reply:
https://developers.google.com/style/word-list?hl=en#letter-symbol

Reply:
Specify what the model and relationships belong to. There are multiple domain models for different features and customers should be able to find them via search